CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+
+If you would like to contribute to the Flutter project,
+we’re happy to have your help! Anyone can contribute, whether you’re new to
+the project or you’ve been around a long time, and whether you self-identify
+as a developer, an end user, or someone who just can’t stand seeing typos.
+
+For information on contributing code to the Flutter SDK, see
+[Contributing to Flutter](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md)
+in the [Flutter SDK](https://github.com/flutter/flutter) repo.
+
+We are also happy to accept your PRs to the
+[Flutter website](https://github.com/flutter/website/) repo.
+
+For ways to get involved in the Flutter community or to learn about us,
+visit the [Flutter community](/community) page.
+
+# Ways to contribute
+
```

Comment:
the "Ways to contribute" section doesn't include the ways that are mentioned in the introduction, such as fixing typos or sending PRs.

Reply:
Addressed.